### PR TITLE
SAP Connector 4.4 Reference Update

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: modules/ROOT/pages/sap/sap-connector-reference-44.adoc [hunk 1]
```diff
@@ -0,0 +1,807 @@
...
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+SAP Connector v4.4
+
+Release Notes: xref:release-notes::connector/sap-connector-release-notes-mule-4.adoc[SAP Connector Release Notes]
+
+The Anypoint Connector for SAP (SAP Connector) provides a class that has an overall configuration for the SAP extension.
```
Comment (george-hoffer) [inline modules/ROOT/pages/sap/sap-connector-reference-44.adoc]: Kim says to not put "The" before the connector name.
Just: Anypoint Connector for SAP (SAP Connector)

File: modules/ROOT/pages/sap/sap-connector-reference-44.adoc [hunk 2]
```diff
@@ -0,0 +1,807 @@
...
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> | The reconnection strategy to use. |  | 
+|===
+
+[[reconnect]]
```
Comment (george-hoffer) [inline modules/ROOT/pages/sap/sap-connector-reference-44.adoc:723]: Replace reconnect and reconnect forever with:
[[reconnect]]
=== Reconnect

[%header,cols="20s,25a,30a,15a,10a"]
|===
| Field | Type | Description | Default Value | Required
| Frequency a| Number | How often to reconnect (in milliseconds). | |
| Count a| Number | The number of reconnection attempts to make. | |
| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
|===

[[reconnect-forever]]
=== Reconnect Forever

[%header,cols="20s,25a,30a,15a,10a"]
|===
| Field | Type | Description | Default Value | Required
| Frequency a| Number | How often in milliseconds to reconnect. | |
| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
|===

File: modules/ROOT/pages/sap/sap-connector-reference-44.adoc [hunk 3]
```diff
@@ -0,0 +1,807 @@
...
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  | 
+|===
+
+
+=== For Configurations
+* <<outbound-config>> 
+
+==== Throws
+* SAP:INVALID_CREDENTIALS_ERROR
```
Comment (george-hoffer) [inline modules/ROOT/pages/sap/sap-connector-reference-44.adoc]: (Global) Alphabetize these

File: modules/ROOT/pages/sap/sap-connector-reference-44.adoc [hunk 4]
```diff
@@ -0,0 +1,807 @@
...
+* <<createTransactionId>> 
+* <<executeQueuedRemoteFunctionCall>> 
+* <<executeSynchronousRemoteFunctionCall>> 
+* <<executeTransactionalRemoteFunctionCall>> 
+* <<getFunction>> 
+* <<send>> 
+
+
+
+== Operations
```
Comment (george-hoffer) [inline modules/ROOT/pages/sap/sap-connector-reference-44.adoc]: Move the list of operations from Supported Operations to == Operations and remove the Supported Operations heading.

File: modules/ROOT/pages/sap/sap-connector-studio.adoc [hunk 5]
```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
 
-Install the SAP Connector and configure the connector in Anypoint Studio. 
+Install SAP Connector and configure the connector in Anypoint Studio.
```
Comment (george-hoffer) [inline modules/ROOT/pages/sap/sap-connector-studio.adoc:7]: You don't install connectors in Mule 4, you add them to a project. Please fix this point in the headings. All Studio 7 connectors are provided in the sense that you don't need to restart Studio to access them -- just set a dependency in the POM file using the internal search feature or using Exchange.

**Review by george-hoffer — APPROVED** [review]

**Review by FilipBogdanSV — APPROVED** [review]

**Review by sorinmare — APPROVED** [review]

**Review by george-hoffer — APPROVED** [review]

**Review by george-hoffer — APPROVED** [review]

**Pull request merged**